tutorials/asr/Voice_Activity_Detection.ipynb

```diff
@@ -657,12 +657,12 @@
                 "We can dramatically improve the time taken to train this model by using Multi GPU training along with Mixed Precision.\n",
                 "\n",
                 "```python\n",
-                "# Mixed precision:\n",
-                "trainer = Trainer(amp_level='O1', precision=16)\n",
-                "\n",
                 "# Trainer with a distributed backend:\n",
                 "trainer = Trainer(devices=2, num_nodes=2, accelerator='gpu', strategy='dp')\n",
                 "\n",
+                "# Mixed precision:\n",
+                "trainer = Trainer(amp_level='O1', precision=16)\n",
```

Comment:
@fayejf amp_level is unused for native amp, remove in future.

Reply:
okay will change it in main later.